lib/flight_scheduler/event_processor.rb

```diff
@@ -190,4 +180,47 @@ def job_step_failed(node_name, job_id, step_id)
     execution.state = 'FAILED'
   end
   module_function :job_step_failed
+
+  def cancel_job(job)
+    case job.job_type
+    when 'JOB'
+      cancel_batch_job(job)
+    when 'ARRAY_JOB'
+      cancel_array_job(job)
+    else
+      Async.logger.error("Can not cancel #{job.job_type} #{job.id}")
```

Comment:
Typo: `Cannot` instead of `Can not`.

Reply:
https://github.com/openflighthpc/flight-scheduler-controller/pull/22/commits/9027e157563ccbc453d6b8b4f33e52af6c224e4a